cg/services/delivery_message/messages/rna_delivery_message.py

```diff
@@ -51,31 +52,23 @@ def __init__(self, store, strategy: RNAUploadMessageStrategy):
         self.strategy = strategy
 
     def create_message(self, cases: list[Case]) -> str:
-        if len(cases) == 1:
-            return self._get_case_message(cases[0])
         return self._get_cases_message(cases)
 
-    def _get_case_message(self, case: Case) -> str:
+    def _get_scout_message_for_case(self, case: Case) -> str:
```

Comment:
```suggestion
    def _get_case_message(self, case: Case) -> str:
```
We do not need to kn ow whether it is for scout or not

Reply:
Don't we?
Now the logic is that for each case, we get a scout message and a file delivery message. This function only gives the part for scout, not for the whole case.

Reply:
I agree, I think it makes sense to mention Scout in the name

Reply:
Will keep it!